### DISCUSSION TITLE
Attn Maintainers Todo List

### DISCUSSION THREAD

**Discussion by ZeroDiscord** [ZeroDiscord/Giveaway#41] [Help needed]
# The following tasks are to be done as of now:

- Upgrade to discord.js v14.x
- Reformat [README.md](https://github.com/ZeroDiscord/Giveaway/blob/master/README.md) 
    - Update GIFs
    - Add new feature updates
    
   All appropriate PRs would be merged instantly 
   
   P.S. Moving all project related discussions ( non - bugs, etc ) to [discussions tab](https://github.com/ZeroDiscord/Giveaway/discussions) issues tab reserved for bugs.

**Comment by ZeroDiscord** [comment]
@Justman10000  @Aim2339 Please note

**Reply by Aim2339** [reply]
Aight 👍
